### PR TITLE
Pytorch lightning sdk example - Missed Notebook added

### PR DESCRIPTION
**Pull request by brianchunkang**
If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [ ] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [ ] Follow the style and grammar rules outlined in the above notebook template.
- [ ] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [ ] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/docs/contributing.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/docs/CODEOWNERS) file under `# Official Notebooks` section, pointing to the author or the author's team.
- [ ] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

If you are opening a PR for `Community Notebooks` under the [notebooks/community](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/community) folder:
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/docs/CODEOWNERS) file under the `# Community Notebooks` section, pointing to the author or the author's team.
- [ ] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/docs/contributing.md#code-quality-checks).

If you are opening a PR for `Community Content` under the [community-content](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/community-content) folder:
- [x] Make sure your main `Content Directory Name` is descriptive, informative, and includes some of the key products and attributes of your content, so that it is differentiable from other content
- [x] The main content directory has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/docs/CODEOWNERS) file under the `# Community Content` section, pointing to the author or the author's team.
- [x] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/docs/contributing.md#code-quality-checks).

### CODE REVIEW

File: notebooks/community/sdk/sdk_pytorch_lightning_custom_container_training.ipynb [hunk 1]
```diff
@@ -0,0 +1,1562 @@
+{
```
Comment (andrewferlitsch) [inline notebooks/community/sdk/sdk_pytorch_lightning_custom_container_training.ipynb:1]: <p>Resnet -&gt; ResNet</p><p>Cifar10 -&gt; CIFAR-10</p><p>Vertex -&gt; Vertex AI</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/269/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (andrewferlitsch) [inline notebooks/community/sdk/sdk_pytorch_lightning_custom_container_training.ipynb:1]: <p>remove TODO</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/269/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Comment (andrewferlitsch) [inline notebooks/community/sdk/sdk_pytorch_lightning_custom_container_training.ipynb:1]: <p>suggest being descriptive. Like "train the model locally"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/269/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Comment (andrewferlitsch) [inline notebooks/community/sdk/sdk_pytorch_lightning_custom_container_training.ipynb:1]: <p>Vertex Training -&gt; Vertex AI</p><p>Custom Container -&gt; custom container</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/269/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Comment (andrewferlitsch) [inline notebooks/community/sdk/sdk_pytorch_lightning_custom_container_training.ipynb:1]: <p>sentence case titles:</p><p> </p><p>Build the custom container</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/269/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Comment (andrewferlitsch) [inline notebooks/community/sdk/sdk_pytorch_lightning_custom_container_training.ipynb:1]: <p>suggest combining into a single code cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/269/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Comment (andrewferlitsch) [inline notebooks/community/sdk/sdk_pytorch_lightning_custom_container_training.ipynb:1]: <p>Drop "then" in title</p><p>Google Cloud Storage -&gt; Cloud Storage</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/269/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>

Comment (andrewferlitsch) [inline notebooks/community/sdk/sdk_pytorch_lightning_custom_container_training.ipynb:1]: <p>suggest removing comments: #ADDED</p><p> </p><p>On GCSFuse conversion, make sure subfolders exist:</p><p> </p><p># GCSFuse conversion</p><p>gs_prefix = 'gs://'</p><p>gcsfuse_prefix = '/gcs/'</p><p>if args.model_dir.startswith(gs_prefix):</p><p>&nbsp;&nbsp;args.model_dir = args.model_dir.replace(gs_prefix, gcsfuse_prefix)</p><p>&nbsp;&nbsp;dirpath = os.path.split(args.model_dir)[0]</p><p>&nbsp;&nbsp;if not os.path.isdir(dirpath):</p><p>&nbsp;&nbsp;&nbsp;&nbsp;os.makedirs(dirpath)</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/269/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>

Comment (andrewferlitsch) [inline notebooks/community/sdk/sdk_pytorch_lightning_custom_container_training.ipynb:1]: <p>Be more specific that the user is testing the training the model within the container image locally.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/269/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='56'/>

Comment (andrewferlitsch) [inline notebooks/community/sdk/sdk_pytorch_lightning_custom_container_training.ipynb:1]: <p>Tensorboard -&gt; TensorBoard</p><p>sentence case the title</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/269/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='65'/>

Comment (andrewferlitsch) [inline notebooks/community/sdk/sdk_pytorch_lightning_custom_container_training.ipynb:1]: <p>sentence case the title</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/269/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='66'/>

Comment (andrewferlitsch) [inline notebooks/community/sdk/sdk_pytorch_lightning_custom_container_training.ipynb:1]: <p>sentence case title</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/269/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='74'/>

Comment (andrewferlitsch) [inline notebooks/community/sdk/sdk_pytorch_lightning_custom_container_training.ipynb:1]: <p>remove code cell, this step is already done above.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/269/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='75'/>

Comment (brianchunkang) [reply notebooks/community/sdk/sdk_pytorch_lightning_custom_container_training.ipynb:1]: <p>Will do</p>

Comment (brianchunkang) [reply notebooks/community/sdk/sdk_pytorch_lightning_custom_container_training.ipynb:1]: <p>Will do</p>

Comment (brianchunkang) [reply notebooks/community/sdk/sdk_pytorch_lightning_custom_container_training.ipynb:1]: <p>Will do</p>

Comment (brianchunkang) [reply notebooks/community/sdk/sdk_pytorch_lightning_custom_container_training.ipynb:1]: <p>Will do</p>

[the remark this answers is not in the document]
Comment (brianchunkang) [reply notebooks/community/sdk/sdk_pytorch_lightning_custom_container_training.ipynb:1]: <p>Will add that in</p>

Comment (brianchunkang) [reply notebooks/community/sdk/sdk_pytorch_lightning_custom_container_training.ipynb:1]: <p>I use this to reset the timestamp for a new run, but the job has a different name so I'll remove it</p>

Comment (brianchunkang) [reply notebooks/community/sdk/sdk_pytorch_lightning_custom_container_training.ipynb:1]: <p>Will do</p>

Comment (brianchunkang) [reply notebooks/community/sdk/sdk_pytorch_lightning_custom_container_training.ipynb:1]: <p>Will do</p>

Comment (brianchunkang) [reply notebooks/community/sdk/sdk_pytorch_lightning_custom_container_training.ipynb:1]: <p>Will do</p>

Comment (brianchunkang) [reply notebooks/community/sdk/sdk_pytorch_lightning_custom_container_training.ipynb:1]: <p>Wanted to highlight the changes from the base code from the PyTorch Lightning docs. Would this work:</p><p>"# CHANGED FROM ORIGINAL SOURCE CODE" or something along these lines</p>

Comment (brianchunkang) [reply notebooks/community/sdk/sdk_pytorch_lightning_custom_container_training.ipynb:1]: <p>Nevermind, it looks cleaner to remove the #ADDED/CHANGED comment. I added the check for the subdirectories as well. Will test now</p>

Comment (brianchunkang) [reply notebooks/community/sdk/sdk_pytorch_lightning_custom_container_training.ipynb:1]: <p>Will do</p>

Comment (brianchunkang) [reply notebooks/community/sdk/sdk_pytorch_lightning_custom_container_training.ipynb:1]: <p>Will do</p>

Comment (andrewferlitsch) [reply notebooks/community/sdk/sdk_pytorch_lightning_custom_container_training.ipynb:1]: <p>Close, missed these two instances:</p><p> </p><pre class="ql-syntax" spellcheck="false">"Your Tensorboard Resource Name or Tensorboard ID" 
</pre><p> </p>

**Review by andrewferlitsch — APPROVED** [review]

**Pull request merged**